Documentation/ContentElements/Images/Index.rst

```diff
@@ -78,18 +80,72 @@ Media adjustments
 Manually specify the width and height of the image in pixels, and apply a
 border.
 
+..  figure:: /Images/ManualScreenshots/ContentElements/TextWithImage/MediaAdjustments.png
+    :alt: Screenshot of the Media Adjustments section of a "Text and Images" content element in TYPO3
+
+    A value of '0' means the size is calculated dynamically.
+
+..  note::
+    This section is only visible when your site package is based on the
+    Site Package Tutorial or on Fluid Styled Content. Site Packages based on
+    the Bootstrap Package have means to dynamically set the size of the images.
+    The images will resize responsively to fill their container.
+
+.. _images-gallery:
+
 Gallery settings
 ----------------
 
 Choose where to position the image in relation to the text.
 
-Images are automatically arranged in columns, two by two. You can change this
+In "Text with Images" content elements you can position the image or images in
+respect to the text here.
+
+Multiple images are automatically arranged in columns, two by two. You can change this
 behaviour with the :guilabel:`Number of Columns` field.
 
+.. _images-behavior:
 
 Behavior
 --------
 
 Use the :guilabel:`Enlarge on Click` setting to enable a lightbox display for the image.
 
 .. include:: /ContentElements/TipKeyboardCommands.rst.txt
+
+.. _images-filelist:
+
+Manage your images in the Filelist
+==================================
+
+You can manage your previously uploaded images in the backend module
+:guilabel:`File > Filelist`:
+
+..  figure:: /Images/ManualScreenshots/ContentElements/TextWithImage/FileList.png
+    :alt: Screenshot of the "Filelist" backend module in TYPO3
+
+    You can view the files as Tiles or List by switching the settings.
+
+TYPO3 uses a file abstraction layer (FAL) you can therefore move images into
+different folders without impending the frontend output of the web page, even if
+they are used in multiple places.
+
+You can move images into folders by drag and drop:
+
+..  figure:: /Images/ManualScreenshots/ContentElements/TextWithImage/FileListDragAndDrop.png
+    :alt: Screenshot of the "Filelist" backend module demonstration Drag and Drop
+
+    Grab a picture with the mouse, drag it to a folder and drop it there
+
+The file abstraction layer will not allow you to delete images that are still in
+use in some content element:
+
+..  figure:: /Images/ManualScreenshots/ContentElements/TextWithImage/FileNotDeleted.png
+    :alt: File not deleted, The file "xxx.png" cannot be deleted since it is still used at the following places
+
+    TYPO3 FAL will prevent you from deleting files that are still in use
+
+First alter or delete all content elements that still use the file, then you can
```

Comment:
Finding such records can be very handy for editor. Should we add a note about how to use the ref info window when clicking the ref-amount link? At the very bottom the is a link to the referenced record.

Reply:
Good point! I think I will add that information to the list module description and link from here